examples/utils.py

```diff
@@ -30,6 +28,11 @@
         ('Verlet chain', 'verlet_chain_demo.py'),
         ('Koch Fractal', 'fractal.py'),
     ])),
+    ('Colors', OrderedDict([
```

Comment:
Just wanted to comment to 👍🏻 the new section.

Reply:
Great, I was getting worried that all the color things are overwhelming the other examples.